### PR TITLE
Created admin page component

### PR DESCRIPTION
**Pull request by dtebo**
Created the admin page component and added the user information table. The user information table is designed to allow the admin to modify and deactivate user accounts.

### CODE REVIEW

File: src/components/Admin/Admin.js [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+        }
+      ],
+      onFilter: (value, record) => record.fname.indexOf(value) === 0,
+      key: 'fname',
+      render: text => <a>{text}</a>,
+    },
+    {
+      title: 'Last Name',
+      dataIndex: 'lname',
+      filters: [
```
Comment (SethMacP) [inline src/components/Admin/Admin.js:26]: Would we be able to type into this at all for searches? 

Can we also lock the table headers on scroll so they are always visible?
(Dont want to scroll to the top to change filters)

File: src/components/Admin/Admin.js [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+];
+
+const data = [
+    {
+      key: '1',
+      fname: 'John',
+      lname: 'Brown',
+      age: 32,
+      address: 'New York No. 1 Lake Park',
+      tags: ['mentor'],
```
Comment (SethMacP) [inline src/components/Admin/Admin.js:104]: I need to read up more on the tags ant design uses. This is very useful!

**Review by SethMacP — COMMENTED** [review]
Great work Darren - I am excited for you to show this off to the stakeholders. It looks great and you made it so quick!

**Comment by dtebo** [comment]
> Great work Darren - I am excited for you to show this off to the stakeholders. It looks great and you made it so quick!

Thank you! I appreciate the feedback. What helped was the prebuilt table component and great documentation about it.

File: src/components/Admin/Admin.js (hunk 2 above)
Comment (dtebo) [reply src/components/Admin/Admin.js:104]: I agree. They did a great job on this component. There are tons of features.

File: src/components/Admin/Admin.js (hunk 1 above)
Comment (dtebo) [reply src/components/Admin/Admin.js:26]: I imagine it would be possible. I will look into that. Thank you for the suggestions.

**Comment by sdkramer** [comment]
I have those same questions as Seth: about adding our search functionality right into this table (headmasters, mentors, mentees); and locking headers on scroll.
I too now will go check out the docs to understand better the table component.
Thanks for your work on this!

**Comment by dtebo** [comment]
> I have those same questions as Seth: about adding our search functionality right into this table (headmasters, mentors, mentees); and locking headers on scroll.
> I too now will go check out the docs to understand better the table component.
> Thanks for your work on this!

You're welcome. Thank you as well for the feedback. Much appreciated!

**Comment by sdkramer** [comment]
Looking over the docs--so far, the antd table looks like it can do a lot of what we want.

**Review by sdkramer — APPROVED** [review]
Code looks clean. No duplicate code.
Thanks for your work!

**Pull request merged**